cocos/scripting/js-bindings/manual/js_module_register.cpp

```diff
@@ -1,7 +1,5 @@
-#ifndef __JS_TEMPLATE_RUNTIME_FRAMEWORKS_RUNTIME_SRC_CLASSES_JS_MODULE_REGISTER_H__
-#define __JS_TEMPLATE_RUNTIME_FRAMEWORKS_RUNTIME_SRC_CLASSES_JS_MODULE_REGISTER_H__
+#include "js_module_register.h"
 
-#include "scripting/js-bindings/manual/ScriptingCore.h"
```

Comment:
Not change .h to .cpp, we add the .cpp, and move the implement from .h to .cpp, so simulator can only include the .h file, we can decrease the function need to be export.
As this change , both simulator and cocostudio can include this .h file to build as dylib.